extensions/etcdsnapshot/etcdsnapshot.go

```diff
@@ -1,8 +1,10 @@
 package etcdsnapshot
 
 import (
+	"context"
```

Comment:
is this needed? its a new addition

Reply:
there's an additional line rm'd here that I couldn't find just by skimming, is the extra removal necessary? just seeing that the line count changes aren't the same as 2.9/2.8